docs/develop/integrations/aws-secrets-manager.md

```diff
@@ -110,6 +110,65 @@ async function run() {
 run();
 ```
 
+  </TabItem>
+  <TabItem value="java" label="Java" default>
+
+```java
```

Comment:
hey this is really cool! thanks for doing it. I would like to show you something about our docs tomorrow morning, I think there might be a better place for us to put this code. I'll ping you and we can find a time to zoom

Reply:
Cool! have also added a new helper on the sdk package examples with similar logic and tested that it worked succesfully. Currently waiting on 
 the previous [PR](https://github.com/momentohq/client-sdk-java/pull/291) on the sdk package to be approved and merged before I cherry-pick the secrets manager commit on my local.

Reply:
@cprice404 so we are holding off on this PR until you get this code into the SDK repo and be able to link back to this?

Reply:
that is what I want to discuss with Pratik, yes.

Reply:
yes am closing this and will generate a new PR with pages just linking to the examples under SDK folders.